### PR TITLE
fix(connect-ui): update Connect UI tabs styling with new design enhancements

### PR DESCRIPTION
**Pull request by gregfromstl**
### TL;DR

Update the styling of Connect UI tabs.

### What changed?

Modified the styles within `Tabs.tsx`, including adding border-radius, setting specific colors and shadows for the selected tab button, and removing the `Line` component.

### How to test?

Verify the updated styling of tabs within the Connect UI. Ensure that the selected tab has distinct styling according to the changes made.

### Why make this change?

To improve the visual appearance and user experience of the Connect UI tabs.

---

---

## PR-Codex overview
This PR updates the styling of Connect UI tabs and improves the layout consistency in the wallet transaction history screens.

### Detailed summary
- Updated styling of Connect UI tabs in `Tabs.tsx`
- Adjusted layout in `BuyTxHistory.tsx` and `WalletTransactionHistory.tsx`
- Added `Spacer` component for spacing consistency
- Renamed active tab from "Purchases" to "Transactions" in `TransactionsScreen.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by joaquim-verges — APPROVED** [review]

**Comment by MananTank** [comment]
Need to make these buttons full-width: 
image

**Review by MananTank — DISMISSED** [review]
Fix purchases tab button width

**Review by MananTank — APPROVED** [review]

**Review by MananTank — CHANGES_REQUESTED** [review]
image

Need to fix styles in light mode

File: packages/thirdweb/src/react/web/ui/components/Tabs.tsx [hunk 1]
```diff
@@ -16,19 +17,33 @@ export default function Tabs({
...
-      <Container flex="row" center="y" style={{ width: "100%" }}>
+      <Container
+        flex="row"
+        center="y"
+        style={{ width: "100%", borderRadius: radius.md, marginTop: 8 }}
+        p="xxs"
+        bg="secondaryButtonBg"
+      >
         {options.map((option) => (
           <button
```
Comment (MananTank) [inline packages/thirdweb/src/react/web/ui/components/Tabs.tsx:28]: Prefer using `Button` component here

File: packages/thirdweb/src/react/web/ui/components/Tabs.tsx [hunk 2]
```diff
@@ -16,19 +17,33 @@ export default function Tabs({
...
               alignItems: "center",
               justifyContent: "center",
               position: "relative",
+              borderRadius: radius.md,
+              color: option === selected ? theme.colors.primaryText : "auto",
+              backgroundColor:
+                option === selected ? theme.colors.modalBg : "transparent",
+              boxShadow:
+                option === selected
+                  ? `0px 0px 10px ${theme.colors.modalBg}`
```
Comment (MananTank) [inline packages/thirdweb/src/react/web/ui/components/Tabs.tsx:45]: Can we remove this shadow? We are not using shadows anywhere else - so this stands out too much

**Review by MananTank — APPROVED** [review]

**Pull request merged**